src/explore-education-statistics-admin/src/pages/release/data/__tests__/ReleaseApiDataSetDetailsPage.test.tsx

```diff
@@ -243,6 +265,30 @@ describe('ReleaseApiDataSetDetailsPage', () => {
       await screen.findByText('Draft version details'),
     ).toBeInTheDocument();
 
+    // Tasks
+
+    expect(
+      screen.getByRole('heading', { name: 'Draft version tasks' }),
+    ).toBeInTheDocument();
+
+    const mapLocationsTask = within(screen.getByTestId('map-locations-task'));
+    expect(
+      mapLocationsTask.getByRole('link', { name: 'Map locations' }),
+    ).toHaveAttribute(
+      'href',
+      '/publication/publication-1/release/release-1/api-data-sets/data-set-id/locations-mapping',
+    );
+    expect(mapLocationsTask.getByText('Incomplete'));
```

Comment:
Should have a `toBeInTheDocument` at the end.

Same applies for 'Map filters' assertion below